src/main/java/org/opensearch/ad/transport/ForwardADTaskTransportAction.java

```diff
@@ -79,33 +80,58 @@ protected void doExecute(Task task, ForwardADTaskRequest request, ActionListener
         DetectionDateRange detectionDateRange = request.getDetectionDateRange();
         String detectorId = detector.getDetectorId();
         ADTask adTask = request.getAdTask();
+        User user = request.getUser();
+        Integer availableTaskSlots = request.getAvailableTaskSLots();
 
         String entityValue = adTaskManager.convertEntityToString(adTask);
 
         switch (adTaskAction) {
+            case APPLY_FOR_TASK_SLOTS:
+                logger.debug("Received APPLY_FOR_TASK_SLOTS action for detector {}", detectorId);
+                adTaskManager.checkTaskSlots(adTask, detector, detectionDateRange, user, ADTaskAction.START, transportService, listener);
+                break;
+            case CHECK_AVAILABLE_TASK_SLOTS:
+                logger.debug("Received SCALE_TASK_SLOTS action for detector {}", detectorId);
```

Comment:
nit: the logged action value doesn't match the case value

Reply:
Good catch, will fix.